### PR TITLE
Drafts flight plan for the 25th of August

### PR DESCRIPTION
**Pull request by JuliaWindmiller1987**
This new flight plan makes use of some cool changes in pyorcestra:
- includes a vertical cross section to illustrate the profile
- outputs a table of the important waypoints

It also includes the LatLon objects in a way that is no longer hard coded but the longitude (if applicable) is calculated based on the corresponding EarthCARE track.

### CODE REVIEW

File: orcestra_book/plans/HALO-20240825a.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    format_version: 0.12
+    jupytext_version: 1.7.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+platform: HALO
+flight_id: HALO-20240825a
+takeoff: "2024-08-25 09:30:00Z"
+landing: "2020-08-22 18:30:00Z"
```
Comment (lkluft) [inline orcestra_book/plans/HALO-20240825a.md:15]: Back to the future!

**Review by lkluft — CHANGES_REQUESTED** [review]

File: orcestra_book/plans/HALO-20240825a.md (hunk 1 above)
Comment (JuliaWindmiller1987) [reply orcestra_book/plans/HALO-20240825a.md]: Changed...

Comment (lkluft) [reply orcestra_book/plans/HALO-20240825a.md]: Now it's only four years off ;)

Comment (JuliaWindmiller1987) [reply orcestra_book/plans/HALO-20240825a.md]: Oh je... Sorry

**Comment by lkluft** [comment]
The build is currently failing because `mindelo` is not defined in the stable release of pyorcestra. I just issued a new release. If you change the `environment.yaml` to require `orcestra>=0.0.15`, the build should work again

**Review by lkluft — APPROVED** [review]

**Comment by lkluft** [comment]
Could you rebase and force-push your branch so that the updated environment is active? ☺️ I guess the report would build now after a merge, but I don't want to merge with failing CI 😬

**Pull request merged**